app/views/denounces/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="row justify-content-center mt-5">
+  <div class="col-xs-12 col-xl-10">
+    <div class="card mb-3 shadow">
+      <div class="card-header">
+        <h3>Edit Denounce</h3>
```

Comment:
I recommend that in a later issue the texts can be added to an i18n internalization file, that can help them to have the texts organized.